### PR TITLE
Get pixel ratio of device from running javascript command in ipykernel

### PR DESCRIPTION
**Pull request by nvaytet**
The `pixel_ratio` can be hard-set in the user's config.
If it is not found in the config, we run some javascript in the kernel to get it from the browser.

### CODE REVIEW

File: docs/scipp-neutron/instrument-view.ipynb [hunk 1]
```diff
@@ -81,7 +81,7 @@
    "source": [
     "You can adjust opacities and create a cut surface to slice your data in 3D using the control buttons below the scene.\n",
     "\n",
-    "It is possible to customize the figure using the usual arguments, as well as adjusting the pixel size, e.g."
+    "It is possible to customize the figure using the usual arguments, e.g."
```
Comment (SimonHeybrock) [inline docs/scipp-neutron/instrument-view.ipynb:84]: Why was this removed? Is it documented elsewhere? I think it will still be frequently used, at least for instrument with non-square pixels.

**Comment by nvaytet** [comment]
@SimonHeybrock does this work on both your monitors?

**Comment by SimonHeybrock** [comment]
It works on my Mac (high res). Will check docs with normal monitor once built.

**Comment by nvaytet** [comment]
This won't really work in the docs because it depends on the system that is building the docs, not the display with which you are viewing the docs.
In addition, when building the docs, the value for the pixel ration that gets passed from the kernel is `None` (see the comment I made about the javascript being run asynchronously, i.e. only once the cell has finished executing).
So in the docs, the scaling will always be 1.

By the way, this also means that this will work **only** if you have your imports in a separate cell from the `plot` command. If they are in the same cell, the pixel ratio will be `None`.
